sdk/storage/Cargo.toml

```diff
@@ -131,23 +131,5 @@ required-features=["blob"]
 name="stream_blob_01"
 required-features=["blob"]
 [[example]]
-name="list_queues"
-required-features=["queue"]
-[[example]]
-name="queue_create"
-required-features=["queue"]
-[[example]]
-name="get_messages"
-required-features=["queue"]
-[[example]]
-name="peek_messages"
-required-features=["queue"]
-[[example]]
-name="delete_message"
-required-features=["queue"]
-[[example]]
-name="put_message"
-required-features=["queue"]
```

Comment:
@johnbatty should these have been added to the new crate?

Reply:
@thovoll I don't think so.  I believe that examples only need to be included in `Cargo.toml` if they need additional flags - in this case to add `required-features=["queue"]`.  In the new crate, this feature flag no longer exists, so we can remove the example sections from `Cargo.toml`.

Let me know if you disagree, or I am missing something.

Reply:
I vaguely remember Cargo complaining when I added examples that were not in Cargo.toml, but it might indeed be because of `required-features`. If you can run the examples using Cargo, we're all set I think :)